internal/interpreter/dynamic_item_tree.rs

```diff
@@ -1210,6 +1210,7 @@ pub(crate) fn generate_item_tree<'id>(
 
     fn property_info_for_type(
         ty: &Type,
+        name: &SmolStr,
```

Comment:
```suggestion
        name: &str,
```

(then you won't need to convert SPECIAL_PROPERTY_MODEL_DATA to a SmolStr)